src/App.vue

```diff
@@ -215,15 +215,21 @@ export default class App extends Mixins(StateMixin) {
       this.flashMessage.open = true
     })
 
-    const legacyFavIcons = document.querySelectorAll("link[rel*='icon'][type='image/png']")
-
-    legacyFavIcons.forEach(item => {
+    const clearNodes = (item: Element) => {
       const parentElement = item.parentElement
 
       if (parentElement) {
         parentElement.removeChild(item)
       }
-    })
+    }
+
+    const legacyFavIcons = document.querySelectorAll("link[rel*='icon'][type='image/png']")
+
+    legacyFavIcons.forEach(clearNodes)
+
+    const legacyThemeColor = document.querySelectorAll("meta[name='theme-color']")
+
+    legacyThemeColor.forEach(clearNodes)
```

Comment:
Maybe something like this would be cleaner here?
```suggestion
    for (const selector of ["link[rel*='icon'][type='image/png']", "meta[name='theme-color']"])
      document.querySelectorAll(selector).forEach(clearNodes)
```

Though not tested for functionality..

Reply:
Following on your comments, I refactored this bit of code!